HW04/ihar_shkuryn/lib/homework.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Homework
+  attr_reader :title, :description, :student, :mentor
+  attr_accessor :notices, :answers
+
+  def initialize(data, mentor)
+    @title = data[:title]
+    @description = data[:description]
+    @student = data[:student]
+    @mentor = mentor
+    @status = data[:status]
+    message = "Student #{@student.name} #{@student.surname} got a new homework "\
```

Comment:
consider moving this logic from initializer. it has to serve for assigning values or some very important logic calls

Reply:
done